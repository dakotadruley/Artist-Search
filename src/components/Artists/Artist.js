import React from 'react';
import PropTypes from 'prop-types';

const Artist = ({ name }) => (
  <h2>{name}</h2>
);

Artist.propTypes = {
  id: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
};

export default Artist;

