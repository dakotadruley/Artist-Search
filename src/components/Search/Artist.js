import React from 'react';
import PropTypes from 'prop-types';

const Artist = ({ name, url }) => (
  <a href={url}>{name}</a>
);

Artist.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
};

export default Artist;

// id like for them to click on the artist and have that url take you somewhere
