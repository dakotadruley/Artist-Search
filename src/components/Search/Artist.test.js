import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist.js';

describe('Artist', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Artist
      name={'dakota superstar'}
      url={'url'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
