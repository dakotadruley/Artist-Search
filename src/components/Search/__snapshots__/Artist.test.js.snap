// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist matches a snapshot 1`] = `
<a
  href="url"
>
  dakota superstar
</a>
`;
